drivers/vfs/driver.go

```diff
@@ -193,7 +193,9 @@ func (d *Driver) create(id, parent string, opts *graphdriver.CreateOpts, ro bool
 	}
 	labelOpts := []string{"level:s0"}
 	if _, mountLabel, err := label.InitLabels(labelOpts); err == nil {
-		label.SetFileLabel(dir, mountLabel)
+		if err := label.SetFileLabel(dir, mountLabel); err != nil {
```

Comment:
Can we fail the entire operation if this fails? Wouldn’t it break on non-SELinux systems? Maybe a debug log is the best we can do here.

Cc: @rhatdan , history is 73617e5e18159e5f791d2860c7857cd5dea31be4 .

Reply:
mountlabel should be "" on a non SELinux system.

SetFileLabel becomes a no-op.